modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AspNetCoreServerCodegen.java

```diff
@@ -407,4 +465,26 @@ private void setAspnetCoreVersion(String packageFolder) {
         }
         additionalProperties.put(COMPATIBILITY_VERSION, compatibilityVersion);
     }
+
+    private void setUseSwashbuckle() {
+        if (isLibrary) {
+            useSwashbuckle = false;
+        }
```

Comment:
Looks like this requires an else block.

Reply:
Added an else and also a warning to indicate that useSwashbuckle default is different when building a library.